src/ciflow-bot.ts

```diff
@@ -0,0 +1,220 @@
+import * as probot from 'probot';
+
+export class CIFlowBot {
+  // Constructor required
+  readonly ctx: probot.Context;
+
+  // Static readonly configurations
+  static readonly allowed_commands: string[] = ['ciflow'];
+  static readonly bot_app_name = 'pytorchbot';
+  static readonly bot_assignee = 'pytorchbot';
+  static readonly event_issue_comment = 'issue_comment';
+  static readonly event_pull_request = 'pull_request';
```

Comment:
I'm not sure what the utility of defining these as constants are, since they're part of the GitHub API and cannot change

Reply:
(I see I disagree with @malfet on this point. Defer to @malfet haha)

Reply:
If the same string literal is used 20 times in the code, there is a chance someone can make a typo in 21st instance and spend hours debugging it. Having it as constant will detect a typo at linter time rather that hunting it at compile time.

And I'm not well versed in TypeScript, but imo it would be better to have them in its own namespace, for example something like:
```
namespace GHAConstants {
   static readonly issue_comment = 'issue_comment';
   static readonly pull_request = 'pull_request';
};
```

Reply:
Yes, I've been consolidating them into class static constants, and since we only have two, probably not worth having a dedicated global namespace just to document two string literals - it adds complexity for the readers. We can split them out when we are targeting more and more events.